.github/workflows/release-cypress-pkg.yaml

```diff
@@ -0,0 +1,59 @@
+name: Publish Cypress Suite Package
+
+on:
+  push:
+    tags:
+      - 'cypress-pkg-v*'
+  workflow_call:
+    inputs:
+      tag:
+        required: false
+        type: string
+    outputs:
+      build-release-cypress-suite-status: 
+        value: ${{ jobs.build.outputs.build-status }}
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./cypress
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.repository == 'rancher/dashboard' && (github.event_name == 'workflow_call' || (github.event_name == 'push' && github.event.ref == 'refs/tags/${{ github.ref_name }}'))
+    outputs:
+      build-status: ${{ job.status }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v6
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v6
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
+
+      - uses: actions/setup-node@v6
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+          scope: '@rancher'
+
+      - name: Install packages
+        run: yarn install --frozen-lockfile
+
+      - name: Build Cypress Package
+        run: yarn build-pkg
+
+      - name: Publish Cypress Package to npm
+        run: yarn publish-pkg
```

Comment:
I think you miss ` --access public`

> Note that for legacy reasons scoped packages are by default published with an access set to restricted (aka "private packages"). This requires you to register for a paid npm plan. In case you simply wish to publish a public scoped package to the registry (for free), just add the --access public flag. This behavior can be enabled by default through the [npmPublishAccess](https://yarnpkg.com/configuration/yarnrc#npmPublishAccess) settings.

https://yarnpkg.com/cli/npm/publish#details